reporting-cli/docs/dev/DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,36 @@
+## Developer Guide
+
+
+
+### SOP for installing and running reporting-cli
+
+1. Checkout this package from version control
+    ```
+    git clone git@github.com:opensearch-project/dashboards-reporting.git
+    cd dashboards-reporting
+    git checkout main
+    ```
+2. Run `npm install` inside `reporting-cli/src`
+3. You can run the below commands inside `reporting-cli/src`
+    ```
+    node index.js --url <url>
+    ```
+    For additional command line parameter options
+    ```
+    node index.js -h
+    ```
+4. Alternatively, you can use npm install to run this command from any directory.
+    ```
+    cd `reporting-cli/
+    npm install -g .
```

Comment:
should install instructions also be in reference manual? (maybe after we publish to npm, it's ok)

also just an idea, should `reporting` executable name be more specific?

Reply:
Any suggestion for alternate executable name? Earlier I thought `opensearch-reports` but wasn't sure if it is short enough

Reply:
i was thinking osd-something but not a convention. there's `opensearch-cli` and `opensearchsql`, but `opensearch-reporting` or `opensearch-reports` is indeed longer, maybe leave it for now